### ISSUE TITLE
Aex9 balances internal server error caused by "Out of gas"

### ISSUE THREAD

**Issue by jyeshe** [aeternity/ae_mdw#267]
This request returns 500:
https://mainnet.aeternity.io/mdw/aex9/balances/ct_2M4mVQCDVxu6mvUrEue1xMafLsoA1bgsfC3uT95F3r1xysaCvE

Locally it shows the reason of "Out of gas":
http://localhost:4000/aex9/balances/ct_2M4mVQCDVxu6mvUrEue1xMafLsoA1bgsfC3uT95F3r1xysaCvE

ct_2M4mVQCDVxu6mvUrEue1xMafLsoA1bgsfC3uT95F3r1xysaCvE
ct_21oSi6uMMcnnyxc1KTP8MReBPVvBQQ9oeoA7aexQBsF2o16HwT
ct_2czjMpbeeds9Geb272UfiEvhQHc8QZM5AM8jngwpqdMnWNDP3h
ct_25JPViJPDPjroiL1wr1ngWJ8pK5ykN7x3LqGMWPCuC6PtLjzKT
ct_2mzKUzT55wz4wYUmH4s3dHf3YBprDLi92uxvEah8fe3BaLfkBU
ct_2bFV4kxtmUeKTF5Eb5STfnAx6UgxzBqTL1pYLF1GBhcmoQAhLf
ct_BmCPDh6TK3KtWdwKEJrSCnrvghbkjekuTTh7B5tLGduF6rTFG
ct_227nuD7YiHL7pJ63FP1CnbhKgU9ueWKP77sA26huE47s31tuqQ
ct_jyw2fRpr6srNwSqE6FJFyZSD1upD91eZ5ZZKZfBhuDmBxTU53
ct_VxFemyM2ML15ApWrFqBKrarVbNfBASPBs1SC1zMuwaWrnrTSy
ct_2GPhKGbRxZaaJtqRTnxLk5UDSNNEtRt3tyWuKVQeoVrerRfhHa
ct_7UfopTwsRuLGFEcsScbYgQ6YnySXuyMxQWhw6fjycnzS5Nyzq
ct_M9yohHgcLjhpp1Z8SaA1UTmRMQzR4FWjJHajGga8KBoZTEPwC
ct_BwJcRRa7jTAvkpzc2D16tJzHMGCJurtJMUBtyyfGi2QjPuMVv
ct_uGk1rkSdccPKXLzS259vdrJGTWAY9sfgVYspv6QYomxvWZWBM
ct_26GAggwrHJLw45vopxu1HA51t6pw4xjzqvq66kh1WPtZjNXMyL

**Comment by marc0olo** [comment]
@jyeshe just for curiosity -> is it clear how to solve that now? :-)

**Comment by jyeshe** [comment]
Yes @marc0olo , gonna explore the dry-run and try to replace the dispatch with it. 
Thank you!

**Comment by jyeshe** [comment]
@marc0olo @thepiwo unfortunately the result was also "Out of gas" with dry-run:

```
{:ok,
 {[
    contract_call_tx: {:ok,
     {:call,
      <<240, 130, 89, 115, 67, 253, 140, 219, 140, 118, 32, 246, 72, 131, 235,
        207, 63, 224, 201, 76, 11, 193, 45, 81, 42, 93, 164, 48, 63, 85, 77,
        200>>,
      {:id, :account,
       <<13, 24, 60, 171, 170, 28, 99, 114, 174, 14, 112, 19, 49, 53, 233, 194,
         46, 149, 172, 14, 114, 22, 38, 51, 153, 136, 58, 149, 27, 56, 30,
         105>>}, 1, 455213,
      {:id, :contract,
       <<133, 131, 168, 71, 25, 30, 125, 215, 20, 116, 235, 252, 117, 187, 174,
         103, 20, 104, 33, 207, 9, 127, 240, 26, 95, 164, 11, 47, 99, 169, 10,
         75>>}, 1000000, 10, "Out of gas", :error, []}}
  ], []}}
```  
  
If I try to reduce more the gas_price (less than `div(min_gas_price(), 1000)`, it results in:
`{:ok, {[contract_call_tx: {:error, :too_low_gas_price}], []}}`

If try to reduce the fee (less than `190 * min_gas_price()`) , gives an error too:
`{:ok, {[contract_call_tx: {:error, :too_low_fee}], []}}`

For:
``` 
    def min_gas_price do
      protocol = :aec_hard_forks.protocol_effective_at_height(1)

      :erlang.max(
        :aec_governance.minimum_gas_price(protocol),
        :aec_tx_pool.minimum_miner_gas_price()
      )
    end
```

**Comment by marc0olo** [comment]
There is definitely a possibility to omit this. Question is how you can achieve this easily with the current way it is implemented. We might have to consult with Dimitar here again.
